rush-plugins/rush-init-project-plugin/src/plopfile.ts

```diff
@@ -27,24 +29,29 @@ import {
   getTemplateNameList,
   getTemplatesFolder,
 } from "./logic/templateFolder";
+import type { ICliParams } from "./init-project";
 
 export interface IExtendedAnswers extends Answers {
+  authorName: string;
+  description: string;
   template: string;
   packageName: string;
   unscopedPackageName: string;
   projectFolder: string;
   shouldRunRushUpdate: boolean;
 }
 
-export default function (plop: NodePlopAPI): void {
+export default function (
+  plop: NodePlopAPI,
+  plopCfg: PlopCfg & ICliParams
+): void {
   const rushConfiguration: RushConfiguration = loadRushConfiguration();
   const monorepoRoot: string = rushConfiguration.rushJsonFolder;
-  const isDryRun: boolean =
-    process.argv.includes("--dry-run") || Boolean(process.env.DRY_RUN);
 
   const hooks: IHooks = initHooks();
   const pluginContext: IPluginContext = {
-    isDryRun,
+    isDryRun: plopCfg.dryRun || Boolean(process.env.DRY_RUN),
+    externalAnswer: typeof plopCfg.answer === "object" ? plopCfg.answer : {},
```

Comment:
Will it be better renaming `externalAnswer` to `cliAnswer`? It maybe more clear to tell it passed from CLI parameter. What do you think?